boards/msbiot/include/cc110x_params.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
A better solution would be to move this file in `drivers/cc110x/include` then using `#ifndef/#define/#endif` it will be possible to set default values in the driver code that can be overridden in the boards. The msba2 could also be updated.
See how it's done with at86rf2xx driver and samr21-xpro/iotlab-m3 boards.

Reply:
Thanks for the hint. I will update it accordingly.

Btw: Does the cc110x work for those boards? Last time I checked the driver didn't work with either msba2 or msbiot. The driver was developed when `spi_acquire` was a rather cheap operation of acquiring a mutex, not setting up the SPI bus. Before the SPI API was changed so that `spi_acquire` also configured the SPI bus, the driver worked on both msba2 and msbiot. So optimizing the driver to call `spi_acquire` only once at the very beginning of every interaction with the hardware and `spi_release` at the very end might not only improve performance, but also fix the issue with msba2 and msbiot boards.